service/src/test/java/bio/terra/workspace/service/workspace/flight/CreateGcpContextFlightV2Test.java

```diff
@@ -32,6 +32,7 @@
 import bio.terra.workspace.service.workspace.GcpCloudContextService;
 import bio.terra.workspace.service.workspace.WorkspaceService;
 import bio.terra.workspace.service.workspace.exceptions.MissingSpendProfileException;
+import bio.terra.workspace.service.workspace.flight.gcp.*;
```

Comment:
please specify the import.

also move flight to flight/gcp

Reply:
curious about the * import - is it against our coding standard? If so, we should add a lint rule. I know IntelliJ has collapsed imports for me before.

Reply:
from java style guide: https://engdoc.corp.google.com/eng/doc/devguide/java/style/index.md?cl=head#s3-source-file-structure

Reply:
Yes, it was automatically imported by IntelliJ. Changed and created https://broadworkbench.atlassian.net/browse/PF-2510